Sources/Commands/SwiftPackageTool.swift

```diff
@@ -749,8 +749,12 @@ extension SwiftPackageTool {
                 help: "The absolute or relative path to output the resolved dependency graph.")
         var outputPath: AbsolutePath?
 
+        @Flag(name: .long,
+              help: "Fetches or updates manifests to get the dependencies if not cached (can be expensive).")
+        var update: Bool = false
```

Comment:
I feel like the default should continue to be true?

Reply:
I am also a bit confused about what this flag is supposed to determine. iiuc, the desired behavior of `show-dependencies` is to resolve dependences when they have not been resolved yet, but to skip it when they are already known/cached, which is just like `swift build` works. is this is how it works now? or does it always  resolve dependences, even if they are already supposed to be known? if so, this may be a bug, and not require a new flag